### PR TITLE
Add OneLogin as a Supported Identity Provider

### LINKED ISSUE #1517: Add OneLogin as a supported IdP
**Request type:**

- [ ] **New document:** Add the `docs-content` label to this issue.
- [x] **Addition to an existing document**: Add the `enhancement` label to this issue.

**What is the purpose of the document/enhancement? Select all that apply:**

- [ ] Document a new feature for an upcoming release. Add the appropriate release version label to this issue.
- [x] Information for an existing feature that isn't yet documented.
- [ ] Additional considerations/ reference material for an existing document.

**Problem/feature that requires documentation:**

@bearsandrhinos Has a great write up on enabling OneLogin via SAML to connect to Astro. Let's add the instructions to our docs.
https://www.notion.so/astronomerio/How-to-enable-OneLogin-SAML-authentication-9ed16e26a1e244debeed11d7d127a42b

**Are you the right person to review a draft of the document for technical accuracy? If not, who is?**

### PR DESCRIPTION
**Pull request by kwcanuck**
Resolves #1517 

@jwitz typically, we don't document third-party applications. However, for users to successfully add OneLogin as an IdP, it was necessary to provide complete step-by-step instructions instead of referencing OneLogin documentation. The procedure is specific to Astronomer.

### CODE REVIEW

File: astro/configure-idp.md [hunk 1]
```diff
@@ -166,4 +167,72 @@ When a user assigned to the application accesses Astro, they will be brought aut
...
+
+Save these values for Step 2.
+
+#### Step 2: Create the OneLogin Astronomer application
+
+1. In the OneLogin administrator dashboard, click **Applications** > **Applications** and then click **Add App**.  
+
+2. In the **Search** field, enter **SAML Custom** and then select **SAML Custom Connector (Advanced)**.
+
+3. In the **Display Name** field, enter **Astronomer** and then click **Save**.
```
Comment (chomipi88) [inline astro/configure-idp.md:193]: Let's have them name it `Astro`

File: astro/configure-idp.md [hunk 2]
```diff
@@ -166,4 +167,72 @@ When a user assigned to the application accesses Astro, they will be brought aut
...
+    | Field name | Value           |
+    | ---------  | -----------------| 
+    | email      | Email            |
+    | firstName  | First Name       |
+    | lastName   | Last Name        |
+    | name       | Name             |
+
+    Select the **Include in SAML assertion** checkbox for every parameter that you add and then click **Save**.
+
+8. Click **SSO** in the left menu, click **View Details** below the **X.509 Certificate** field and then click **Download**.
```
Comment (chomipi88) [inline astro/configure-idp.md:216]: Include detail to set SAML Signature Algorithm to SHA-256
https://www.notion.so/astronomerio/How-to-enable-OneLogin-SAML-authentication-9ed16e26a1e244debeed11d7d127a42b#7cdc5489aa4c4d6080c381cc378dd6fb

**Comment by kwcanuck** [comment]
> Add OneLogin to the list here image

@chomipi88 - good catch. OneLogin has been added to the list.

File: astro/configure-idp.md (hunk 1 above)
Comment (kwcanuck) [reply astro/configure-idp.md:193]: @chomipi88 - this has been updated.

File: astro/configure-idp.md (hunk 2 above)
Comment (kwcanuck) [reply astro/configure-idp.md:216]: @chomipi88 - this task is not completed by users. It is completed by Astronomer Support. In step 2, users are instructed to capture the X.509 Certificate and the SAML 2.0 Endpoint (HTTP) URL:

image

In step 3, users are instructed to provide the  X.509 Certificate and the SAML 2.0 Endpoint (HTTP) information to Astronomer support:

image

This format is the same used for Okta and Azure.

Comment (chomipi88) [reply astro/configure-idp.md:217]: This is indeed performed by the customer in the OneLogin console to ensure they export the correct certificate to send to us.
https://www.notion.so/astronomerio/How-to-enable-OneLogin-SAML-authentication-9ed16e26a1e244debeed11d7d127a42b#7cdc5489aa4c4d6080c381cc378dd6fb

**Comment by chomipi88** [comment]
lgtm

**Review by jwitz — APPROVED** [review]
No further suggestions, really clean instructions here!

**Comment by kwcanuck** [comment]
From @chomipi88 2022-11-28:

_Please continue holding. My front end resource is out on sick leave so this has taken a back seat for now._

**Comment by chomipi88** [comment]
@kwcanuck good to go

**Review by chomipi88 — APPROVED** [review]

File: astro/configure-idp.md [hunk 3]
```diff
@@ -168,6 +170,69 @@ When a user assigned to the application accesses Astro, they will be brought aut
...
+#### Step 4: Assign users to your OneLogin Astro application
+
+1. In the OneLogin administrator dashboard, click **Applications** > **Applications** and then click **Astro**.
+
+2. Click **Users** in the left menu.
+
+3. Make sure that all users who are using Astro are assigned to the Astronomer application.
+
+    When a user assigned to the application accesses Astro, they are automatically signed in to OneLogin after entering their email in the Cloud UI.
+
```
Comment (jwitz) [inline astro/configure-idp.md:235]:
```suggestion
</TabItem>

```

File: astro/configure-idp.md [hunk 4]
```diff
@@ -239,7 +306,6 @@ From here, Astronomer support will finalize your organization's integration with
 Assign users from your organization to your new application. See [Managing user groups](https://docs.pingidentity.com/bundle/pingcentral-19/page/awu1616596133840.html).
 
 When a user assigned to the application accesses Astro, they are automatically signed in to Ping Identity after entering their email in the Cloud UI.
```
Comment (jwitz) [inline astro/configure-idp.md:308]:
```suggestion
When a user assigned to the application accesses Astro, they are automatically signed in to Ping Identity after entering their email in the Cloud UI.

```

**Pull request merged**